app/src/main/AndroidManifest.xml

```diff
@@ -6,6 +6,32 @@
   <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
   <uses-permission android:name="android.permission.INTERNET" />
   <uses-permission android:name="android.permission.WAKE_LOCK" />
+  <compatible-screens>
```

Comment:
Can you please point to the reference Android documentation for this as a comment? It'd be helpful for both the review & future reviewing.

I'm also thinking that we might need tests for this (to guard against unexpected screen sizes being added in the future), but it'll depend a bit on the documented Android behavior.

Reply:
@BenHenning  https://developer.android.com/guide/topics/manifest/compatible-screens-element , this is the reference i used.